INSTALL.md

```diff
@@ -73,8 +77,8 @@ matching your compiler, and download all patches!
   - <http://math-atlas.sourceforge.net>
   - <https://www.tacc.utexas.edu/research-development/tacc-software/gotoblas2>
 
-If compiling with OpenMP support then it is recommended to use a non-threaded
-version of BLAS. In particular if compiling with MKL and using OpenMP you must
+Although CP2K is OpenMP parallelized it is recommended to use a non-threaded
+version of BLAS. In particular if compiling with MKL you must
```

Comment:
Hmm, I think we should add add a note here stating that we also require the BLAS/LAPACK calls/library to be OpenMP-threadsafe (especially OpenBLAS).

Reply:
Good point. How should we word this? Also, doesn't threadsafe usually imply that it's threaded?

Reply:
> Good point. How should we word this?

Please note that the BLAS/LAPACK implementation used by CP2K needs to be OpenMP thread-safe. Examples are: the sequential variant of the Intel MKL, the Cray libsci, the OpenBLAS OpenMP variant and the reference BLAS/LAPACK packages.

> Also, doesn't threadsafe usually imply that it's threaded?

No, see the reference LAPACK/BLAS implementation (just hope someone verified that at some point), the sequential MKL.

Reply:
Sounds good. I've adopted your paragraph.